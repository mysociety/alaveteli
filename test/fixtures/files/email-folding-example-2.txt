Preface to the message which we are not interested in

-----------------------------------------------------------------------------------------
Important message about cheese
-----------------------------------------------------------------------------------------

Actual footer that contains the word confidential
