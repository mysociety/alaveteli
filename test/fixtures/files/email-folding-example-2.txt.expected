Preface to the message which we are not interested in

-----------------------------------------------------------------------------------------
Important message about cheese
